test/ClangModules/objc_bridging_generics.swift

```diff
@@ -283,9 +291,11 @@ extension AnimalContainer {
 
 extension PettableContainer {
   func doesntUseGenericParam(_ x: T, _ y: T.Type) {
-    _ = type(of: x).init(fur: x).other()
+    // TODO: rdar://problem/27796375--allocating entry points are emitted as
```

Comment:
Sure, I could move them into separate tests for the time being. I wanted to make sure we still test that the other statements in that function don't trigger the error message.